source/data-api.txt

```diff
@@ -12,32 +12,53 @@ Atlas Data API
    Custom HTTPS Endpoints </data-api/custom-endpoints>
    Authenticate Data API Requests </data-api/authenticate>
    Data API Examples </data-api/examples>
+   Data Formats </data-api/data-formats>
 
 .. contents:: On this page
-   :local:
-   :backlinks: none
    :depth: 2
-   :class: singlecol
 
-The Atlas Data API is a secure, serverless HTTPS API that lets you work with
-data stored in a linked data source or run custom application logic. You
-don't need to install any database drivers or opinionated libraries to
-work with the Data API. Instead, you send requests like this from an
-HTTPS client:
+The Data API is a managed service that lets you securely work with
+data stored in Atlas using standard HTTPS requests. You can use the Data
+API to connect to MongoDB Atlas from any platform that supports HTTPS,
+including:
 
-.. code-block:: bash
+- Web browsers
+- Servers
+- CI/CD pipelines
+- Serverless & Edge compute environments
+- Mobile applications
+- Internet-Of-Things devices
+
+The Data API is not a direct connection to your database. Instead, the
```

Comment:
Nit: I think this might help set the stage better either combined with the first sentence or as a second sentence.  Feels like we're splitting the definition in two here.